docker-compose.yaml

```diff
@@ -145,22 +86,24 @@ services:
     restart: always
     image: httpd:2.4
     volumes:
-      - ${AIRFLOW_PROJ_DIR:-.}/weather:/usr/local/apache2/htdocs
-  
+      - ${AIRFLOW_PROJ_DIR}/weather:/usr/local/apache2/htdocs
+
   postgres:
     image: postgres:13
     environment:
       POSTGRES_USER: airflow
       POSTGRES_PASSWORD: airflow
       POSTGRES_DB: airflow
     volumes:
-      - postgres-db-volume:/var/lib/postgresql/data
+      - ${AIRFLOW_PROJ_DIR}/db:/var/lib/postgresql/data
     healthcheck:
       test: ["CMD", "pg_isready", "-U", "airflow"]
       interval: 10s
       retries: 5
       start_period: 5s
     restart: always
+    ports:
+      - 5432:5432
```

Comment:
this enables you to connect to the airflow postgres, which was helpful for debugging settings. but in practice this might actually be annoying if we're trying to develop an ETL against a local instance of VZ or Moped, which also want port `5432`?

Reply:
that's a great point that you can always attach to the container and `psql` into it. i have removed this port binding from the compose b/c we agree it will be more annoying than it's worth.